modules/external-auth-fields.adoc

```diff
@@ -17,7 +17,36 @@ The following table lists all available OIDC provider parameters for direct auth
 |Description
 
 |`claimMappings`
-|Configures the rules to be used by the Kubernetes API server for translating claims in a JWT token, issued by the identity provider, to a cluster identity.
+|Configures the rules to be used by the Kubernetes API server for translating claims in a JSON web token (JWT), issued by the identity provider, to a cluster identity.
+
+|`claimMappings.extra`
+|An optional field for configuring the mappings used to construct the extra attribute for the cluster identity. When omitted, no extra attributes will be present on the cluster identity. Key values for extra mappings must be unique. A maximum of 32 extra attribute mappings may be provided.
+
+|`claimMappings.extra.key`
+a|A required field that specifies the string to use as the extra attribute key. The following restrictions apply:
+
+* Key must be a domain-prefix path (e.g 'example.org/foo').
+* Key must not exceed 510 characters in length.
+* Key must contain the '/' character, separating the domain and path characters.
+* Key must not be empty.
+* The domain portion of the key (string of characters before the '/') must be a valid RFC1123 subdomain.
+* It must not exceed 253 characters in length.
+* It must start and end with an alphanumeric character.
+* It must only contain lower case alphanumeric characters and '-' or '.'.
+* It must not use the reserved domains, or be subdomains of, "kubernetes.io", "k8s.io", and "openshift.io".
+* The path portion of the key (string of characters after the '/') must not be empty and must consist of at least one alphanumeric character, percent-encoded octets, '-', '.', '_', '~', '!', '$', '&', ''', '(', ')', '*', '+', ',', ';', '=', and ':'.
+* It must not exceed 256 characters in length.
```

Comment:
Seen from `oc explain authentication.spec.oidcProviders.claimMappings.extra`, blank line separation is shown:
```
The domain portion ...
It must not exceed 253 characters in length

The path portion ...
It must not exceed 256 characters in length
```
So, with blank line separation, it reads good.

But seen from the preview page, it is shown as:

- The domain portion ...
- It must not exceed 253 characters in length
- The path portion ...
- It must not exceed 256 characters in length
Two different lines of "It must not exceed ..." are in parallel at first glance. This doesn't read good. Please replace the "It" with "the xxx portion" or use same separation to avoid the reading confusion.